ai/stt/service.py

```
@@ -1,134 +1,237 @@
+    def load_model(self, model_name: Optional[str] = None) -> None:
+        """Whisper 모델 로드"""
+        model_to_load = model_name or self.model_name
 
         try:
-            logger.info(f"Loading Whisper model: {model_name}")
-            self.model = whisper.load_model(model_name, device=self.device)
+            logger.info(f"Loading Whisper model: {model_to_load}")
+            self.model = whisper.load_model(model_to_load, device=self.device)
+            self.model_name = model_to_load
             logger.info("Whisper model loaded successfully")
         except Exception as e:
```

Ruff (0.13.2)

Do not catch blind exception: `Exception`

(BLE001)